docs/encyclopedia/temporal-nexus-overview.mdx

```diff
@@ -0,0 +1,118 @@
+This means you don’t have to use Nexus RPC directly, only the Temporal SDK along with a Temporal Service.
+
+Similar to executing an Activity, for a Nexus Operation:
+
+1. Caller Workflow will schedule a Nexus Operation using the Temporal SDK, which sends a `ScheduleNexusOperation` command in the existing `RespondWorkflowTaskCompleted` RPC which is recorded as `NexusOperationScheduled` in the caller’s Workflow history.
+2. Nexus handler Worker uses the Temporal SDK to `PollNexusTaskQueue` and process the Nexus request.
+3. Nexus handler Worker sends a `RespondNexusTaskCompleted` via the SDK, with the result of the Nexus request, which may be one of: Started, Completed, Failed, TimedOut, Canceled,  see Nexus Operation Events for more info
+4. Caller Workflow then processes the Nexus Operation event via `PollWorkflowTaskQueue`.
+
+See also: [Nexus Architecture on GitHub](https://github.com/temporalio/temporal/blob/main/docs/architecture/nexus.md).
+
+## Key Capabilities
+
+Nexus support is available in Temporal Cloud, OSS Server, and Temporal SDKs:
+
+- Workers that handle Nexus Operations, listening on a Namespace and Task Queue
+- Nexus API registry, to create Nexus Endpoints that forward requests to a target Namespace and Task Queue, so others can use Nexus services via a Nexus endpoint.
+- Start Nexus operations from Temporal Workflows across team & Namespace boundaries
+- Observe and troubleshoot end-to-end execution
+
+Temporal Cloud provides enhanced security, observability, and multi-region routing:
+
+- Multi-region routing from Caller to destination Namespace in a Temporal Cloud account
+- Control who can manage a Nexus Endpoint with a new Nexus Endpoint Admin permission
+- Control who can use a Nexus Endpoint within an account, using Namespace allowlists
+- Observe Nexus audit logs and observability metrics via cloud integrations
+
+#### Integrated Temporal SDK experience
+
+- Run Nexus Services in a Temporal Worker:
+  - Abstract and expose a subset of Temporal primitives for others to use
+  - Can also execute arbitrary code
+  - Typically the same Worker as the underlying Temporal primitives
+  - Same queue-based Worker architecture, for Worker fleet load balancing
+  - No separate deployment for a bespoke service wrapper
+  - Polling architecture enables workers to run in a broad range of runtime environments
+- Use Nexus Services from a Workflow
+  - Start a Nexus Operation from a Temporal Workflow
+
+#### Built-in Nexus Machinery
+
+- Built-in support in Temporal Server and Temporal Cloud
+- Connects Nexus callers and handlers through Nexus Endpoints
+- Routes Nexus calls across team and Namespace boundaries
+- Automatic retries
+- Integrated into Workflow event history for end-to-end debugging
+- Supports metrics, traces, and logs
```

Comment:
Can we turn these sections into paragraphs instead of these bullet lists? This doesn't feel very polished to me and lacks context.

Reply:
Agreed. These were taken out and replaced with a new overview. I do want to try to avoid short bullet point lists.